lgn-messages/src/types/mod.rs

```diff
@@ -1,445 +0,0 @@
-use std::fmt::Display;
-use std::fmt::Formatter;
-
-use derive_debug_plus::Dbg;
-use serde_derive::Deserialize;
-use serde_derive::Serialize;
-use thiserror::Error;
-
-use crate::routing::RoutingKey;
-
-pub mod experimental;
-pub mod v1;
-
-const REQUIRED_STAKE_SMALL_USD: Stake = 98777;
-const REQUIRED_STAKE_MEDIUM_USD: Stake = 98777;
-const REQUIRED_STAKE_LARGE_USD: Stake = 169111;
-
-/// A keyed payload contains a bunch of bytes accompanied by a storage index
-pub type KeyedPayload = (String, Vec<u8>);
-
-pub trait ToKeyedPayload {
-    fn to_keyed_payload(&self) -> KeyedPayload;
-}
-
-pub type HashOutput = [u8; 32];
-
-#[derive(Clone, Debug, Deserialize, Serialize)]
-pub enum TaskType {
-    TxTrie(experimental::tx_trie::WorkerTask),
-    RecProof(experimental::rec_proof::WorkerTask),
-    V1Preprocessing(v1::preprocessing::WorkerTask),
-    V1Query(v1::query::WorkerTask),
-    V1Groth16(v1::groth16::WorkerTask),
-}
```

Comment:
Removed the dead code for the experimental variants, and moved this to v1 so it is version defined.